exp-player/addon/components/exp-mood-questionnaire.js

```diff
@@ -0,0 +1,361 @@
+                }
+            },
+            "/childStats": {
+                templates: {
+                    container: bootstrapContainer
+                }
+            },
+            "/childStats/schedule/time": {
+                templates: {
+                    control: bootstrapControl
+                }
+            },
+            "/childStats/wakeup": {
+                templates: {
+                    control: bootstrapControl
+                }
+            },
+            "/childStats/eat": {
+                templates: {
+                    control: bootstrapControl
+                }
+            },
+            "/childStats/previousActivities": {
+                templates: {
+                    control: bootstrapControl
+                }
+            }
+        }
+    }
+};
+
+export default ExpFrameBaseComponent.extend({
+    layout: layout,
+    type: 'exp-mood-questionnaire',
+    meta: {
+        name: 'ExpMoodQuestionnaire',
+        description: 'Mood questionnaire for Lookit.',
+        parameters: {
+            type: 'object',
+            properties: {
+                id: {
+                    type: 'string',
+                    description: 'A unique identifier for this item'
+                },
+                form: {
+                    type: 'jsonschema',
+                    description: 'JSON-schema defining this item\'s form',
+                    default: defaultSchema
+                }
+            },
+            required: ['id']
+        },
+        data: {
+            type: 'object',
+            properties: {
+                formData: {
+                    type: 'object'
+                }
+            }
+        }
+    },
+    formSchema: Ember.computed('form', {
+        get() {
+            var root = this;
+            var newOptions = this.get('form.options');
+            newOptions.form = {
+                buttons: {
+                    update: {
+                        title: 'Continue',
+                        type: 'button',
+                        styles: 'btn btn-default'
+                    }
+                }
+            };
+            return {
+                schema: this.get('form.schema'),
+                options: newOptions,
+                view: this.get('form.view')
+            };
+        },
+        set(_, value) {
+            this.set('formSchema', value);
+            return value;
+        }
+
+    }),
+    formData: [],
+    formActions: Ember.computed(function() {
+        var root = this;
+        return {
+            update: function () {
+                this.refreshValidationState(true);
+                if (this.isValid(true)) {
+                   root.set('formData', this.getValue());
+                   root.actions.next();
```

Comment:
This needs to be `root.actions.next.apply(root);` for this to work (the `this` context is wrong otherwise).

Also looks like these lines may be indented 3 spaces and not 4?